Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "catppuccin-egui"
-version = "5.1.1"
+version = "5.1.2"
```

We use release-please to control the versioning of this repository, which is driven by the PR title (i.e. `feat` for minor, `fix` for patch)

Please revert this change.

```suggestion
version = "5.1.1"
```

---

Maybe comment could be added to Cargo.toml or contributing guide added to repository to avoid future problems!

Thanks for the fix, commited the change :)